postman_collection/Test Airlock.postman_collection.json

```diff
@@ -1,78 +1,136 @@
 {
-    "info": {
-        "_postman_id": "3431c32a-55db-4b8c-89f0-89b29cb914a2",
-        "name": "Test Airlock",
-        "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
-    },
-    "item": [
-        {
-            "name": "Add Item",
-            "request": {
-                "method": "POST",
-                "header": [],
-                "body": {
-                    "mode": "raw",
-                    "raw": "{\r\n    \"id\": 1,\r\n    \"name\": \"KB9\"\r\n}",
-                    "options": {
-                        "raw": {
-                            "language": "json"
-                        }
-                    }
-                },
-                "url": {
-                    "raw": "http://localhost:8000/item",
-                    "protocol": "http",
-                    "host": ["localhost"],
-                    "port": "8000",
-                    "path": ["item"]
-                }
-            },
-            "response": []
-        },
-        {
-            "name": "Get Item",
-            "request": {
-                "method": "GET",
-                "header": [],
-                "url": {
-                    "raw": "http://localhost:8000/item/1",
-                    "protocol": "http",
-                    "host": ["localhost"],
-                    "port": "8000",
-                    "path": ["item", "1"]
-                }
-            },
-            "response": []
-        },
-        {
-            "name": "Error Request",
-            "request": {
-                "method": "DELETE",
-                "header": [],
-                "url": {
-                    "raw": "http://localhost:8000/item/2",
-                    "protocol": "http",
-                    "host": ["localhost"],
-                    "port": "8000",
-                    "path": ["item", "2"]
-                }
-            },
-            "response": []
-        },
-        {
-            "name": "Private topic",
-            "request": {
-                "method": "GET",
-                "header": [],
-                "url": {
-                    "raw": "http://localhost:8000/ping",
-                    "protocol": "http",
-                    "host": ["localhost"],
-                    "port": "8000",
-                    "path": ["ping"]
-                }
-            },
-            "response": []
-        }
-    ]
-}
+	"info": {
+		"_postman_id": "bdf33e40-e866-4a23-a4fc-b70c98bbecda",
+		"name": "Test Airlock",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Add Item",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
```

Comment:
we don't need to set the bearer token in each request. In postman, you can right-click the collection, click edit and set the token once here, for all requests 
![image](https://user-images.githubusercontent.com/219256/120236204-bf11e200-c229-11eb-8eb4-3243afd80097.png)